app/src/main/java/org/jellyfin/androidtv/ui/SettingsPopup.kt

```diff
@@ -0,0 +1,72 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.PopupWindow
+import android.widget.TextView
+import org.jellyfin.androidtv.databinding.SettingsPopupBinding
+import org.jellyfin.androidtv.util.Utils
+
+class SettingsPopup(
+	context: Context?,
+	anchor: View,
+	audioChangedListener: ValueChangedListener<Long>?,
+	subtitleChangedListener: ValueChangedListener<Long>?
+) {
+	val popupWindow: PopupWindow?
+	private val mAnchor: View
+	private val mAudioDelaySpinner: NumberSpinnerView
+	private val mSubtitleDelaySpinner: NumberSpinnerView
+	private val mAudioDelaySpinnerTxt: TextView
```

Comment:
No "m" prefixes in new files

Reply:
Also just use `binding.xxx` instead of assigning those to variables in the init block

Reply:
> No "m" prefixes in new files

Any specific coding standards you use? I found [your board](https://github.com/jellyfin/jellyfin-androidtv/projects/15) mentioned in a previous topic. But that doesn't go into detail of variable naming etc.

Reply:
Basically name it to whatever makes the most sense, normally without any prefixes/suffixes. Variables use camelCase while constants (in companion objects) use SCREAMING_SNAKE_CASE